File: CSharpServer/Qipai4Core/Qipai4Core/net/silverfoxserver/core/protocol/ClientAction.cs
/*
* SilverFoxServer: massive multiplayer game server for Flash, ...
* VERSION:3.0
* PUBLISH DATE:2015-9-2 
* GITHUB:github.com/wdmir/521266750_qq_com
* UPDATES AND DOCUMENTATION AT: http://www.silverfoxserver.net
* COPYRIGHT 2009-2015 SilverFoxServer.NET. All rights reserved.
* MAIL:521266750@qq.com
*/
using System;
using System.Collections.Generic;
using System.Text;
//
using net.silverfoxserver.core.logic;

namespace net.silverfoxserver.core.protocol
{
    /// <summary>
    /// 与客户端QiPaiClient 的Send函数字符串相对应
    /// 
    /// const 相当于是 static readonly
    /// </summary>
    public static class ClientAction
    {
        public static readonly string verChk = "verChk";

         /**
         *  整个系统的数据库类型，全局
         */
        public static readonly String loadDBType = "loadDBType";

        public static readonly String hasReg = "hasReg";

        public static readonly string reg = "reg";

        public static readonly string login = "login";
        
        public static readonly string listRoom = "listRoom";

        public static readonly string listModule = "listModule";

        public static readonly string pubMsg = "pubMsg";

        //声音聊天
        public static readonly string pubAuMsg = "pubAuMsg";

        public static readonly string joinRoom = "joinRoom";

        public static readonly string joinReconnectionRoom = "joinReconnectionRoom";

        public static readonly string autoJoinRoom = "autoJoinRoom";

        public static readonly string autoMatchRoom = "autoMatchRoom";

        /// <summary>
        /// 加载好友列表
        /// </summary>
        public static readonly string loadB = "loadB";

        /// <summary>
        /// 加载空闲用户列表
        /// </summary>
        public static readonly string loadD = "loadD";

        /// <summary>
        /// 刷新金币
        /// </summary>
        public static readonly string loadG = "loadG";

        /// <summary>
        /// 下注
        /// </summary>
        public static readonly string setBetVars = "setBetVars";

        //房间变量更新
        //准备，游戏动作，走棋等
        public static readonly string setRvars = "setRvars";

        //让收信人自动打开的邮件系统
        public static readonly string setMvars = "setMvars";

        /// <summary>
        /// 模块系统变量更新
        /// </summary>
        public static readonly string setModuleVars = "setModuleVars";

        /// <summary>
        /// 离开房间，回到大厅
        /// </summary>
        public static readonly string leaveRoom = "leaveRoom";

        /// <summary>
        /// 离开房间，转入重新匹配界面
        /// </summary>
        public static readonly string leaveRoomAndGoHallAutoMatch = "leaveRoomAndGoHallAutoMatch";

        /// <summary>
        /// 关闭网页，模拟
        /// </summary>
        public static readonly string sessionClosed = "sessionClosed";

        /// <summary>
        /// 心跳协议
        /// </summary>
        public static readonly string heartBeat = "heartBeat";

        /// <summary>
        /// 
        /// </summary>
        public static readonly string loadChart = "loadChart";



    }
}

File: JavaServer/Qipai4Core/src/net/silverfoxserver/core/protocol/ClientAction.java
/*
 * SilverFoxServer: massive multiplayer game server for Flash, ...
 * VERSION:3.0
 * PUBLISH DATE:2015-9-2 
 * GITHUB:github.com/wdmir/521266750_qq_com
 * UPDATES AND DOCUMENTATION AT: http://www.silverfoxserver.net
 * COPYRIGHT 2009-2015 SilverFoxServer.NET. All rights reserved.
 * MAIL:521266750@qq.com
 */
package net.silverfoxserver.core.protocol;

/** 
 与客户端QiPaiClient 的Send函数字符串相对应
 
 const 相当于是 static final readonly
*/
public class ClientAction
{
	public static final String verChk = "verChk";
        
        /**
         *  整个系统的数据库类型，全局
         */
        public static final String loadDBType = "loadDBType";
        
        public static final String hasReg = "hasReg";

	public static final String reg = "reg";

	public static final String login = "login";

	public static final String listRoom = "listRoom";

	public static final String listModule = "listModule";

	public static final String pubMsg = "pubMsg";

	//声音聊天
	public static final String pubAuMsg = "pubAuMsg";

	public static final String joinRoom = "joinRoom";

	public static final String joinReconnectionRoom = "joinReconnectionRoom";

	public static final String autoJoinRoom = "autoJoinRoom";

	public static final String autoMatchRoom = "autoMatchRoom";

	/** 
	 加载好友列表
	*/
	public static final String loadB = "loadB";

	/** 
	 加载空闲用户列表
	*/
	public static final String loadD = "loadD";

	/** 
	 刷新金币
	*/
	public static final String loadG = "loadG";

	/** 
	 下注
	*/
	public static final String setBetVars = "setBetVars";

	//房间变量更新
	//准备，游戏动作，走棋等
	public static final String setRvars = "setRvars";

	//让收信人自动打开的邮件系统
	public static final String setMvars = "setMvars";

	/** 
	 模块系统变量更新
	*/
	public static final String setModuleVars = "setModuleVars";

	/** 
	 离开房间，回到大厅
	*/
	public static final String leaveRoom = "leaveRoom";

	/** 
	 离开房间，转入重新匹配界面
	*/
	public static final String leaveRoomAndGoHallAutoMatch = "leaveRoomAndGoHallAutoMatch";

	/** 
	 关闭网页，模拟
	*/
	public static final String sessionClosed = "sessionClosed";

	/** 
	 心跳协议
	*/
	public static final String heartBeat = "heartBeat";

	/** 
	 
	*/
	public static final String loadChart = "loadChart";
        
        /**
         * 排行榜
         * 
         */
        public static final String loadTopList = "loadTopList";
        



}
